cli/dcoscli/node/main.py

```diff
@@ -65,12 +65,214 @@ def _cmds():
             function=_ssh),
 
         cmds.Command(
+            hierarchy=['node', 'snapshot', 'create'],
+            arg_keys=['<nodes>'],
+            function=_snapshot_create),
+
+        cmds.Command(
+            hierarchy=['node', 'snapshot', 'delete'],
+            arg_keys=['<snapshot>'],
+            function=_snapshot_delete),
+
+        cmds.Command(
+            hierarchy=['node', 'snapshot'],
+            arg_keys=['--list', '--status', '--cancel', '--download'],
+            function=_snapshot_manage),
+
+        cmds.Command(
             hierarchy=['node'],
             arg_keys=['--json'],
-            function=_list),
+            function=_list)
     ]
 
 
+def _snapshot_manage(list_snapshots, status, cancel, snapshot_download):
+    """
+    Manage snapshots
+
+    :param list_snapshots: a list of available snapshots
+    :type  list_snapshots: bool
+    :param status: show snapshot job status
+    :type  status: bool
+    :param cancel: cancel snapshot job
+    :type  cancel: bool
+    :param snapshot_download: download a snapshot
+    :type  snapshot_download: string
+    :return: process return code
+    :type: int
+    """
+    if list_snapshots:
+        snapshot_response = _snapshot_get(
+            '/system/health/v1/report/snapshot/list/all')
```

Comment:
Do we have a set way to inform end-users of this? If not, I suggest adding a general way to check the metadata endpoint for the $VERSION for a given request, and determining if this CLI works with that $VERSION of DC/OS (or something like that).